packages/notifier/README.md

```diff
@@ -70,17 +85,10 @@ consume(subscription);
 // eventually prints
 // non-final a
 // non-final b
-// finished
+// the iteration finished
```

Comment:
Point taken. However, I changed the code to match "the iteration finished" because I think it helps to make very clear there's a difference between non-final and final values (even if there really isn't at this level, since either can be a single word or a phrase). It's similar to having "non-final a" as opposed to just "a".